sdk/testcommon/Azure.Management.Network.2020_04/src/Generated/NetworkManagementClient.cs

```diff
@@ -77,8 +77,8 @@ public NetworkManagementClient(string subscriptionId, Uri endpoint, TokenCredent
         /// <summary> Returns an instance of BastionHostsOperations. </summary>
         public virtual BastionHostsOperations BastionHosts => new BastionHostsOperations(_clientDiagnostics, _pipeline, _subscriptionId, _endpoint);
 
-        /// <summary> Returns an instance of ServiceOperations. </summary>
-        public virtual ServiceOperations Service => new ServiceOperations(_clientDiagnostics, _pipeline, _subscriptionId, _endpoint);
+        /// <summary> Returns an instance of AzureManagementNetworkOperations. </summary>
+        public virtual AzureManagementNetworkOperations AzureManagementNetwork => new AzureManagementNetworkOperations(_clientDiagnostics, _pipeline, _subscriptionId, _endpoint);
```

Comment:
Where are we getting this name from?

Reply:
I am guessing it is removing the whitespaces in the title.
Eg: "Azure Communication Chat Service" is becoming "AzureCommunicationChatServiceRestClient". In which case, I wonder if we should set the titles differently.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-09-21-preview2/communicationserviceschat.json#L4

Reply:
Forgot to regenerate code for this, it was generated by older version.